praveenkumar1410.md

```diff
@@ -0,0 +1,58 @@
+
+## FUTURE SCOPE
+It is a miniature model for a voice controlled wheel chair( For differently abled people). They can easily controll the wheel chair though commands. In further development it can also function using neural commands(Direct impulse from the brain).
+
+## Plan
+
+1. Collect all the parts I need to make this project.The principle this toy concept was based on was voice recognition technology. 
+2. Make a car with the components( DC motor 12V,  micro motors, grippy wheels, battery, Texas Instruments Dual H-Bridge motor drivers L293D).
+3. Connect it with the Bluetooth module. 
+4. Connect wires to the arduino uno board.
+5. Connect Arduino with the Motor Shield and then connect all the wires coming from motors and servo.
+6. Assemble all items.
+7. Program the machine with arduino IDE to verify if everything is working as expected.
+8. Prepare the machine for the first testrun by using ardiuino software.
+9. Test the custom made dashboard by running the machine again using the dashboard generated g-code.
+10. Complete the project if everything work as expected.
+11. Connect the Raspberry pi4. The script on the Raspberry Pi that handles the processing of requests is written in Python.When the python script is activated, this opens the link between the Arduino Uno and the Raspberry Pi, thus enabling the car's motors and lights.This script also opens a communication link to a servo connected to pin 18 on the Raspberry Pi. This servo controls the turning mechanism in the car. The script uses PubNub to retrieve Alexa's requests and feeds these requests into a series of if statements. Depending on the command, the script will either send the command to the Arduino Uno through serial communication, tell the servo to turn the car, or return an unknown command error.
+
+## Raspberri Pi4 SET UP
+
+1.) Download the Raspberry Pi K'nex Car Control image.
+
+2.) flash the 16 gb sd card with the image.
+
+3.) Plug the monitor (via HDMI), mouse, keyboard, Arduino Uno (via USB), and WiFi adapter (if applicable) into the Raspberry Pi.
+
+4.) Power on the Raspberry Pi
+
+5.) When the Raspberry Pi finishes booting, it will ask you to connect to WiFi.
+
+## Existing Project
+
+https://create.arduino.cc/projecthub/mathwuy/voice-controlled-k-nex-car-f1b867?ref=similar&ref_id=443067&offset=4
```

Comment:
for some reason whenever i go to this link it redirects to https://projecthub.arduino.cc/ :( would you be able to share a downloaded pdf or smth of the page so we could view it as reference?

Reply:
is it the same as https://www.hackster.io/mathwuy/voice-controlled-k-nex-car-f1b867?